src/Game.js

```diff
@@ -0,0 +1,48 @@
+function Game() {
+  this._total_score = 0;
+  this._frame_score = 0;
```

Comment:
Because you have frame score, you might be able to write a function which checks if the frame score is 10 and if it is a strike. Means you would be able to increase the number of goes you get from the bonus

Reply:
yes! have slightly ignored the strike on the 10th frame - was thinking I could just have something clever on the front end but didn't get that far - agree a function would have worked better!